build.sh

```diff
@@ -53,6 +54,21 @@ git config --global user.name "$commit_username"
 git config --global user.email "$commit_email"
 echo '::endgroup::'
 
+echo '::group::Updating pkgver'
+if [ "$update_pkgver" = "true" ]; then
+  echo 'Running `makepkg -od` to update pkgver'
+  mkdir -p /tmp/makepkg
+  cp "$pkgbuild" /tmp/makepkg/PKGBUILD
+  (
+    cd /tmp/makepkg;
+    makepkg -od;
+  )
+  pkgbuild=/tmp/makepkg/PKGBUILD
+else
+  echo 'Not updating pkgver'
+fi
+echo '::endgroup::'
```

Comment:
Instead of creating this group (`echo '::group::Updating pkgver'`) regardless of whether user chooses to update `pkgver`, I think it would be better if this group doesn't appear when `update_pkgver` is `false`.